commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/offloaded/OffloadedProperties.scala

```diff
@@ -1,11 +1,14 @@
 package com.lynbrookrobotics.potassium.commons.drivetrain.offloaded
 
 import com.lynbrookrobotics.potassium.commons.drivetrain.twoSided.TwoSidedDriveProperties
+import com.lynbrookrobotics.potassium.control.offload.EscConfig
 import com.lynbrookrobotics.potassium.units.Ratio
 import squants.space.{Degrees, Length}
-import squants.{Angle, Dimensionless, Time}
+import squants.{Angle, Dimensionless}
 
 trait OffloadedProperties extends TwoSidedDriveProperties {
+  implicit val escConfig: EscConfig[Length]
```

Comment:
Hmm implicit seems a bit sketchy here since it'll only be brought into scope by a wild card import. Maybe keep it explicit?